AppCenterReactNativeShared/ios/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'AppCenterReactNativeShared' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for AppCenterReactNativeShared
+  pod 'AppCenter'
```

Comment:
We use `pod 'AppCenter', '~> 2.3.0'` with a version number in DemoApp to allow testing pre-release private native ios sdk. It's more flexible that way. Maybe we should do the same here.